pygmt/src/wiggle.py

```diff
@@ -0,0 +1,96 @@
+"""
+wiggle - Plot z=f(x,y) anomalies along tracks.
+"""
+from pygmt.clib import Session
+from pygmt.helpers import build_arg_string, fmt_docstring, kwargs_to_strings, use_alias
+
+
+@fmt_docstring
+@use_alias(
+    B="frame",
+    D="position",
+    G="color",
+    J="projection",
+    R="region",
+    T="track",
+    U="timestamp",
+    V="verbose",
+    W="pen",
+    X="xshift",
+    Y="yshift",
+    Z="scale",
+    c="panel",
+    i="columns",
+    p="perspective",
+)
+@kwargs_to_strings(R="sequence", c="sequence_comma", i="sequence_comma", p="sequence")
+def wiggle(self, x=None, y=None, z=None, data=None, **kwargs):
+    r"""
+    Plot z=f(x,y) anomalies along tracks.
+
+    Takes a matrix, (x,y,z) triplets, or a file name as input and plots z as a
+    function of distance along track.
+
+    Must provide either ``data`` or ``x``/``y``/``z``.
+
+    Full parameter list at :gmt-docs:`wiggle.html`
+
+    {aliases}
+
+    Parameters
+    ----------
+    x/y/z : float or 1d arrays
+        The x and y coordinates, or arrays of x and y coordinates of the
+        z data point or an array of z data points.
+    data : str or 2d array
+        Either a data file name or a 2d numpy array with the tabular data.
+        Use parameter ``columns`` to choose which columns are x, y, z,
+        respectively.
```

Comment:
Just a note here, we'll rename it to `incols` instead.

Reply:
Noted.